backend/models/notifications.py

```
@@ -0,0 +1,21 @@
+from sqlalchemy import Column, Integer, ForeignKey, String, Boolean, DateTime
+from datetime import datetime, timezone
+from sqlalchemy.orm import relationship
+from database import Base
+from zoneinfo import ZoneInfo
```

Ruff (0.8.2)

`zoneinfo.ZoneInfo` imported but unused

Remove unused import: `zoneinfo.ZoneInfo`

(F401)